app/templates/index.html

```diff
@@ -1,40 +1,75 @@
 {% extends "base.html" %}
-{% block title %}Index{% endblock %}
+{% block title %}Match mentees to mentors{% endblock %}
 {% block head %}
     {{ super() }}
 {% endblock %}
 {% block content %}
 
-    <h1 class="home--heading">Mentor matcher</h1>
-    <p class="lead">Welcome to the mentor matcher.</p>
+<div class="grid-row">
+  <header class="page--header">
+	  <div class="grid-column-full">
+			<span class="page--caption">Tools</span>
+		  <h1 class="page--heading">Mentor matcher</h1>
+	  </div>
+	  <div class="grid-column-two-thirds">
+		  <p class="page--excerpt">This tool helps you to match mentors to mentees as part of the Civil Service LGBT+ mentoring programme.</p>
+	  </div>
+  </header>
+</div>
 
-	<hr>
+<div class="grid-row">
+  <div class="grid-column-two-thirds">
+		<details>
+			<summary>About this tool</summary>
+				<p>Explanatory text about the took can go here...</p>
```

Comment:
I think there's a spelling error here?

What kind of explanatory text are you thinking to include here?

Reply:
We'd need to write this after the tool was finished; but my thinking was that we'd include some instructions for how to use the tool. 

It might not be necessary depending on the upload workflow, but it's something I've done on other tools available here:
https://www.civilservice.lgbt/tools

Happy to remove if you prefer for now?

Reply:
Those tools are all very cool!

I was thinking more like you see on GOV.UK services. Something like

## To use this service, you'll need:
- two files, named mentors.csv and mentees.csv. The column headers should be the same as our template (link to template)

Does that sounds sensible?

Reply:
Ah now that's given me a better idea!
Shall we add a 'start page'?

So the flow would be:

1. `Start page` (to use this service, you'll need, etc..) – click a big 'Start' button to go to ...
2. `Matcher page` with everything else already baked in

What do you think?

For now, I'll remove this chunk of code and we can debate this as a separate feature!

Reply:
p.s. how does one add a page?!